### DISCUSSION TITLE
Generate the blazonry for a completed design

### DISCUSSION THREAD

**Discussion by mdalypgh** [Azgaar/Armoria#180] [Ideas]
I really like the tool but I struggle to document the design using blazonry language, terms, sequence, etc. I created a design I really like but I decided to test how I would describe it on Drawshield. I failed but worked on the terms, order, etc and got close. Not right but close. My question is if Amoria has the information to translate the graphic design into the words. It feels like a big effort but it would be a really interesting addition to this really cool app.

**Comment by Azgaar** [comment]
Hello. No, Amoria has not feature like that. You can try GhatGPT/Claude to generate blazon based on image, but it can only handle simple ones.

**Reply by mdalypgh** [reply]
Thanks for the quick reply and the suggestion. I'll try it out!
